lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -139,6 +137,20 @@ class LineChartPainter extends AxisChartPainter<LineChartData>
     }
   }
 
+  void clipToBorder(ui.Canvas canvas, ui.Size size,) {
```

Comment:
Functions defined according to the usage of them in the paint() method,
please reorder its definition.

Reply:
I don't get it, it's the first function invoked within paint method and it's also declared right after paint

Reply:
You're right.